activerecord/CHANGELOG.md

```diff
@@ -24,6 +24,11 @@
 
 ## Rails 5.0.0 (June 30, 2016) ##
 
+*   AM::Dirty's changed? methods now return an actual boolean, never `nil`, as in 4.2.
+    https://github.com/rails/rails/pull/24515
```

Comment:
True and realized in Rafael's updated comment that "Fixes #" is preferred and has precedent in the changelogs 😁